js/foundation.tabs.js

```diff
@@ -65,7 +65,6 @@ class Tabs {
 
       $tabContent.attr({
         'role': 'tabpanel',
-        'aria-hidden': !isActive,
```

Comment:
>  I guess if default behavior is display:none that is enough to tell the browser these are hidden...

Yes sir, i thinked about it already and that's why i removed .... i think is good to merge if i am not missing something